profiles/base.yaml

```diff
@@ -822,6 +822,7 @@ theurgy_exp_threshold: 33
 theurgy_bead_carving_tool:
 theurgy_use_prayer_mat: false
 theurgy_prayer_mat_room:
+pray_to_chadatru: false
```

Comment:
Could you also remove `pray_at_chadatru_altar` on line 260?

Reply:
Sure thing